### PR TITLE
resource/ecs_service: Add support for customTimeout on delete

### PR DESCRIPTION
**Pull request by stack72**
Deletes can take a long time depending on how long the drain of the
service takes. This can lead to errors like:

```
 timeout while waiting for state to become 'INACTIVE' (last state: 'DRAINING', timeout: 10m0s)
```

This provides the option for a user to override the delete timeout

### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" comments, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Release note for [CHANGELOG](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md):

```release-note

```

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

**Comment by stack72** [comment]
Hi @anGie44 

I see this has been force pushed recently - does this mean it's going to land shortly? I mean this has been outstanding for 8 months with no response and now force-pushed..

Thanks

Paul

**Comment by anGie44** [comment]
hi @stack72, the force-push pertains to changes in `master`, (hopefully) not direct changes made to this PR's branch, as that was unintended. apologies for any confusion! if the PR is still in a good state on your end, I can provide my review comments.

**Comment by stack72** [comment]
Wait, you *overwote my* changes to the PR without telling me this was the case?

**Comment by stack72** [comment]
ah ok, it was because you were updating the PR from master :) Got it!

**Comment by anGie44** [comment]
i'm actually confused myself because I hadn't looked or had knowledge of this PR until you commented 😅 ... interesting that this PR got looped into that force-push..

**Comment by anGie44** [comment]
Might also relate to errors reported in  #4852
```
* aws_ecs_cluster.ecs: ClusterContainsContainerInstancesException: The Cluster cannot be deleted while Container Instances are active or draining.
```

**Comment by stack72** [comment]
Wait, you are also confused? Do you need me to close and reopen a PR so I can be merged?

**Comment by anGie44** [comment]
@stack72, closing and reopening shouldn't be necessary. I don't see any merge/branch-related conflicts reported here. I was just commenting on the git behavior that caught me by surprise

File: website/docs/r/ecs_service.html.markdown [hunk 1]
```diff
@@ -166,6 +166,13 @@ In addition to all arguments above, the following attributes are exported:
 * `iam_role` - The ARN of IAM role used for ELB
 * `desired_count` - The number of instances of the task definition
 
+## Timeouts
+
+`aws_ecs_service` provides the following
+[Timeouts](/docs/configuration/resources.html#timeouts) configuration options:
```
Comment (anGie44) [inline website/docs/r/ecs_service.html.markdown:172]: Nit: to direct user to existing section
```suggestion
[Timeouts](/docs/configuration/resources.html#operation-timeouts) configuration options:
```

**Pull request merged**